cachito/workers/tasks/gomod.py

```diff
@@ -46,6 +46,7 @@ def fetch_gomod_source(request_id, dep_replacements=None):
     env_vars = {
         "GOCACHE": {"value": "deps/gomod", "kind": "path"},
         "GOPATH": {"value": "deps/gomod", "kind": "path"},
+        "GOSUMDB": {"value": "off", "kind": "literal"},
     }
     env_vars.update(config.cachito_default_environment_variables.get("gomod", {}))
```

Comment:
What is the purpose of those `cachito_default_environment_variables`? Would it make sense to put this in there instead?

Reply:
The vars that are doing a somewhat similar task for npm are in the `cachito_default_environment_variables`, so it would make sense to hav ethis one there as well.

Reply:
@chmeliik Yes, you are right! I just pushed changes.

Reply:
For the record, I'll write my understanding of the two mechanisms.

`env_vars` as defined directly in the Cachito code base, is meant to capture any environment variables that must be set and are not configurable. For example, `GOPATH=deps/gomod`, cannot hold a different value because this maps to where within the tarball the go module dependencies are found. Changing this value means changing how the source tarball is constructed.

`cachito_default_environment_variables` has a similar purpose but it's meant for items that could be configured. See the default NPM values for example. Adding `GOSUMBDB=off` to this makes sense because it depends on the pipeline environment.

Hopefully this makes it clearer!